CallWithDoxDialer/DoxDialerCaller.m

```diff
@@ -36,8 +35,71 @@ +(id _Nonnull)shared {
 }
 
 
-#pragma mark Lazy Properties
+#pragma mark Methods
+-(void)dialPhoneNumber:(nonnull NSString *)phoneNumber {
+    if (self.isDialerInstalled) {
+        NSString *urlEscapedPhoneNumber = [phoneNumber stringByAddingPercentEncodingWithAllowedCharacters:[NSCharacterSet URLHostAllowedCharacterSet]];
+        NSURL *launchDialerURL = [NSURL URLWithString:[NSString stringWithFormat:@"/call?targetNumber=%@", urlEscapedPhoneNumber]
+                                        relativeToURL:self.dialerSchemeURL];
+        [UIApplication.sharedApplication openURL:launchDialerURL];
+    }
+    else {
+        [UIApplication.sharedApplication openURL:self.openDialerInAppStoreURL];
+    }
+}
+
+
+
+#pragma mark Icons
+-(nonnull UIImage *)dialerIcon {
+    if(!_dialerIcon) {
+        NSBundle *libraryBundle = [NSBundle bundleForClass:[DoxDialerCaller class]];
+        NSURL *assetsBundleUrl = [libraryBundle URLForResource:@"CallWithDoxDialer" withExtension:@"bundle"];
+        NSBundle *assetsBundle = [NSBundle bundleWithURL:assetsBundleUrl];
+        
+        _dialerIcon = [UIImage imageNamed:@"doximity-dialer-icon"
+                                 inBundle:assetsBundle
+            compatibleWithTraitCollection:nil];
+    }
+    return _dialerIcon;
+}
+
+-(nonnull UIImage *)dialerIconAsTemplate {
+    if(!_dialerIconAsTemplate) {
+        _dialerIconAsTemplate = [self.dialerIcon imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+    }
+    return _dialerIconAsTemplate;
+}
+
 
+/**
+ Generates the dialer icon using the given color.
+ This call is relatively expensive, so if you expect to make it many times
+ (e.g. for each cell in a table), consider the following alternatives:
+ - cache the returned image for a color you expect to reuse.
+ - use `dialerIconAsTemplate` along with a (reused) tinted view.
+ 
+ @param color The color to fill the icon.
+ @return The Doximity Dialer icon, filled with the given color.
+ */
+-(nonnull UIImage *)dialerIconInColor:(nonnull UIColor *)color {
```

Comment:
i feel like it might be a little confusing/misleading. i feel like if they need to do this they can do it themselves